File: Hash or Objects/Question 12/unique_element.rb
def unique_elements(arr)
    hash_elements = {}
    arr.each { |ele| hash_elements[ele] = true }
    return hash_elements.keys
  end
  
  print unique_elements(['a', 'b', 'a', 'a', 'b', 'c']) #=> ["a", "b", "c"]
  puts

File: Hash or Objects/Question 12/unique_element.js
function unique_element(arr){
    let result  = {};
  
    for(let i =0; i<arr.length; i++){
      if(!result[arr[i]]){
        result[arr[i]] = 0;
      }
      result[arr[i]]++;
    }
    return Object.keys(result);
  }
  
  console.log(unique_element(['a', 'b', 'a', 'a', 'b', 'c']))